parm/ufs/fv3/diag_table

```diff
@@ -1,80 +1,80 @@
 "fv3_history",    0,  "hours",  1,  "hours",  "time"
 "fv3_history2d",  0,  "hours",  1,  "hours",  "time"
-"ocn%4yr%2mo%2dy%2hr", @[FHOUT_OCNICE],  "hours",  1,  "hours",  "time",  @[FHOUT_OCNICE],  "hours",  "@[SYEAR] @[SMONTH] @[SDAY] @[CHOUR] 0 0"
-"ocn_daily%4yr%2mo%2dy",  1,  "days",   1,  "days",   "time",  1,  "days",   "@[SYEAR] @[SMONTH] @[SDAY] @[CHOUR] 0 0"
+"@[MOM6_OUTPUT_DIR]/ocn%4yr%2mo%2dy%2hr", @[FHOUT_OCNICE],  "hours",  1,  "hours",  "time",  @[FHOUT_OCNICE],  "hours",  "@[SYEAR] @[SMONTH] @[SDAY] @[CHOUR] 0 0"
```

Comment:
Can we add this to the atmospheric fields as well? Both for uniformity and so we can introduce some more order to the run directory.

Reply:
No can't do.
I have opened an issue in fv3atm repository to add that feature. See https://github.com/NOAA-EMC/fv3atm/issues/822